assets/assetpack.def

```diff
@@ -151,6 +152,7 @@
 < https://cdnjs.cloudflare.com/ajax/libs/jquery-ujs/1.2.1/rails.js
 < javascripts/disable_animations.js [mode==test]
 < https://raw.githubusercontent.com/sorich87/bootstrap-tour/6a1028fb562f9aa68c451f0901f8cfeb43cad140/build/js/bootstrap-tour.min.js
+< https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js
```

Comment:
Is it really worth it? I know we're already locked in with our jQuery usage anyways but still I'd like to keep external dependencies to a minimum.

Reply:
But I checked and the most important js resources and css are external
https://code.jquery.com/jquery-3.3.1.js
https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.js
https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css

I didn't found a 3dresources directory or something similar to put them all.
However if the idea is to move this to a local location, wouldn't be better to create a new PR to move them all?

Reply:
I don't care whether the dependency is listed as an external resource or bundled.

My point was that introducing yet another jQuery-based dependency should be avoided if it is not worth it. Setting and reading a cookie is [doable with a few lines of plain JavaScript](https://www.w3schools.com/js/js_cookies.asp).

The concern here is that jQuery is kind of old fashioned and frameworks like Boostrap and the web community in general are moving away from it. So I'd prefer not to add an extra dependency for the pure sake of doing it "jQuery-style".

Reply:
Fixed. Now the $.cookie is replaced by a pair of functions